src/graph/graphcontext.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _GRAPHCONTEXT_H
+#define _GRAPHCONTEXT_H
+
+#include <assert.h>
+#include "../redismodule.h"
+#include "graph.h"
+#include "../index/index.h"
+#include "../stores/store.h"
+
+// This would really belong in store.h, but I'm putting it here momentarily for visiility
+typedef struct {
+  LabelStore *store;
+  char *storename;
+  LabelStoreType type;
+} StoreHandle;
```

Comment:
This is not a mandatory addition, but it would reduce the number of keyspace lookups.